package.json

```diff
@@ -35,7 +35,6 @@
     "lib/",
     "LICENSE",
     "HISTORY.md",
-    "SECURITY.md",
```

Comment:
Hm, this is interesting. I think it is alright to remove the file from the bundle as long as all the ones we do this for also have a repository or homepage linking to the repository. The should, and this one does, but just want to call it out for folks reviewing the other PRs.